### PR TITLE
Setup Uvicorn log level

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Setup Uvicorn log level

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library

### CODE REVIEW

File: runner.py [hunk 1]
```diff
@@ -73,4 +73,5 @@ def configure_gradio_ui_envs() -> None:
             ssl_keyfile=config.ols_config.tls_config.tls_key_path,
             ssl_certfile=config.ols_config.tls_config.tls_certificate_path,
             ssl_keyfile_password=config.ols_config.tls_config.tls_key_password,
+            log_level="debug",
```
Comment (bparees) [inline runner.py:76]: what's the default level?  is this increasing the verbosity or decreasing it?

also seems like this should be configured via another field in https://github.com/openshift/lightspeed-service/blob/main/ols/app/models/config.py#L559-L560

Comment (tisnik) [reply runner.py:76]: it is basically decreasing # messages in logs

**Comment by bparees** [comment]
/hold

discussed in slack, this sets the loglevel for uvicorn, so we need a way to control it or we won't be able to enable these messages on a customer cluster for debug purposes.

**Pull request closed without merging**